src/constants/editors.ts

```diff
@@ -1,3 +1,12 @@
+import Atom from "../editors/atom";
+import SublimeText2 from "../editors/sublime-text-2";
+import SublimeText3 from "../editors/sublime-text-3";
+import Vim from "../editors/vim";
+import Chrome from "../editors/chrome";
+import Eclipse from "../editors/eclipse";
+import AndroidStudio from "../editors/android-studio";
+import VSCode from "../editors/vscode";
+
 export const ANDROIDSTUDIO = "ANDROIDSTUDIO";
```

Comment:
These were here before your changes, but let's remove them and use editor classes directly.